dashboard/test/factories/plc.rb

```diff
@@ -0,0 +1,56 @@
+FactoryGirl.allow_class_lookup = false
```

Comment:
IMO, that movement should be part of the debt in moving the PLC namespace into the PD namespace, happening at the same time. As long as they are separate namespaces, I feel the factories should be separate.